static/js/containers/VideoDetailPage.js

```diff
@@ -125,6 +125,7 @@ class VideoDetailPage extends React.Component<*, void> {
         video={video}
         cornerFunc={this.updateCorner}
         selectedCorner={videoUi.corner}
+        useYouTube={video.is_public && video.youtube_id !== null}
```

Comment:
do we think this is every likely to be set in the UI or something similar? if not, I'd suggest we just compute this value from `video` inside of the component, instead of passing it in as a prop (it's a little more indirect this way IMO)